include/core/JamomaInterpolator.h

```diff
@@ -108,6 +108,41 @@ namespace Jamoma {
                 return x1 + delta * (x2-x1);
             }
 		};
+                
+        /** Allpass interpolation
+         Testing shows this algorithm will become less accurate the more points it computes between two known samples.
+         Also, because it uses an internal history, the reset() function should be used when switching between non-continuous segments of sampled audio data.
+         @param x0		Unused sample value
+         @param x1		Sample value at prior integer index
+         @param x2		Sample value at next integer index
+         @param x3		Unused sample value
+         @param delta 	Fractional location between x1 (delta=0) and x2 (delta=1) @n
+                        Be aware that delta=1.0 may not return the exact value at x2 given the nature of this algorithm.
+         @return			The interpolated value
+         */
+        template<class T>
+        class Allpass : Base {
+        public:
+            static const int 	delay = 1;
+            T last_out = T(0.0);
```

Comment:
If it is a private member then it should be mCamelCase (you noted that you saw this at the last moment).  It should be `private`, right?  Also since this is filter nomenclature it should be named `mY1` because it the output from 1 sample in the past.